tests/lib/rules/prefer-const.js

```diff
@@ -309,7 +309,7 @@ ruleTester.run("prefer-const", rule, {
         },
         {
             code: "let {a = 0, b} = obj, c = a; b = a;",
-            output: null,
+            output: "const {a = 0, b} = obj, c = a; b = a;",
```

Comment:
It looks like this result will have a runtime error as `b` is reassigned after being declared `const`. Are you sure this is the right approach?

Reply:
Sounds good @kaicataldo